### DISCUSSION TITLE
tailwind-merge v2

### DISCUSSION THREAD

**Discussion by tmarnet** [heroui-inc/tailwind-variants#139] [Ideas]
Hi 👋 

I'd like to know if there are there any plans to support [`tailwind-merge` v2](https://github.com/dcastil/tailwind-merge/releases/tag/v2.0.0) and up ?

Currently `tailwind-variants` only installs the latest `1.x.x` version of `tailwind-merge`.

Thanks!

**Comment by ohearonkg** [comment]
As a follow up to wanting all the goodness from the latest tailwind merge (e.g `size`, `min-w` and `min-h`) to be understood by tailwind variants (e.g. under the hood tailwind merge for merging), I'm also wondering if there is a way to pass a configuration created by the `extendTailwindMerge` function and or the `createTailwindMerge` function?

**Comment by ohearonkg** [comment]
@tmarnet - Looks like there's a draft PR for this... 🤞 

https://github.com/nextui-org/tailwind-variants/pull/148
